tests/Doctrine/Tests/Common/Annotations/PsrCachedReaderTest.php

```diff
@@ -220,6 +220,28 @@ public function testReaderIsNotHitIfCacheIsFresh(): void
         );
     }
 
+    public function testReaderDoesNotCacheIfFileDoesNotExistSoLastModificationCannotBeDetermined(): void
```

Comment:
Nice test. Can you please repeat this test for the `CachedReader` as well?

Reply:
New test added.